DH_Vehicles/Classes/DH_Sdkfz105Transport.uc

```diff
@@ -78,8 +78,8 @@ defaultproperties
     //AmmoIgnitionProbability=0.2  // 0.75 default; 20mm ammo is unlikely to explode
     //TurretDetonationThreshold=5000.0 // increased from 1750
     //above properties dont compile
-    DamagedEffectHealthFireFactor=0.9
-    EngineHealth=50
+    DamagedEffectHealthFireFactor=0.8
+    EngineHealth=100

```

Comment:
Didn't you want to increase the health of the vehicle, not the engine health?

Reply:
> Didn't you want to increase the health of the vehicle, not the engine health?

nah the engine health was needed to be raised, a small burst of a tommy gun could take it out.

**i should have stated clearly that i wanted to change the engine health not the health pool of the vehicle**

reasoning behind, the german sdkfz251 (halftrack) has 150 health, so me and wolf found it reasonable to bring up the overall engine health to 100, as well as this, it stops the vehicle getting blown up by a couple of probing shots by a garand and should hopefully make it a bit more viable